contracts/vortex/CarbonVortex.sol

```diff
@@ -81,7 +125,22 @@ contract CarbonVortex is ICarbonVortex, Upgradeable, ReentrancyGuardUpgradeable,
      * @dev performs contract-specific initialization
      */
     function __CarbonVortex_init_unchained() internal onlyInitializing {
-        _setRewardsPPM(100_000);
+        // set rewards ppm to 5000
+        _setRewardsPPM(5000);
+        // set price reset multiplier to 2x
+        _setPriceResetMultiplier(2);
+        // set min token sale amount multiplier to 4x
+        _setMinTokenSaleAmountMultiplier(4);
+        // set price decay half-life to 12 hours
+        _setPriceDecayHalfLife(12 hours);
+        // set target token price decay half-life to 12 hours
+        _setTargetTokenPriceDecayHalfLife(12 hours);
+        // set target token price decay half-life to 10 days
+        _setTargetTokenPriceDecayHalfLifeOnReset(10 days);
+        // set initial target token sale amount to 100 eth
+        _setMaxTargetTokenSaleAmount(100 ether);
```

Comment:
This assumes that the target token is ETH, while it's actually configurable. Perhaps it should be something like `100 * 10 ** _targetToke.decimals()`?

Reply:
Something to keep in mind here is that some early-phase (2017ish) token contracts have implemented `DECIMALS` or `Decimals` instead of `decimals`, as far as I remember.

Reply:
these values can be configured by the admin on deployment on each chain

Reply:
My point is that the defaults can be buggy themselves

Reply:
changed to `100 * 10 ** _targetToken.decimals()`